pyquil/external/rpcq.py

```diff
@@ -0,0 +1,113 @@
+from typing import Dict, List, Union, Optional, Any
+from typing_extensions import Literal
+from pydantic import BaseModel, Field
+from rpcq.messages import TargetDevice
+
+# TODO, we can move this into the RPCQ repository.
```

Comment:
This shouldn't be merged with this TODO - we can either

1. Actually make this change to rpcq and depend on it here
2. Make an issue here in Pyquil to make the change to rpcq

Your call.

Reply:
Issue created https://github.com/rigetti/pyquil/issues/1307